CHANGELOG.rst

```diff
@@ -254,6 +258,10 @@ Removed
   playbook. The relevant inventory variables have been renamed, check the
   :ref:`upgrade_notes` for details.
 
+- The `resources__group_name` variable has been removed in favor of using
```

Comment:
You need to use double quotes (``) here to have the variable name written in monospace.